src/montagu_deploy/config.py

```diff
@@ -68,9 +80,31 @@ def __init__(self, path, extra=None, options=None):
             "admin": self.admin_ref,
             "contrib": self.contrib_ref,
             "static": self.static_ref,
+            "db_migrate": self.db_migrate_ref,
         }
 
     def build_ref(self, dat, section):
         name = config.config_string(dat, [section, "name"])
         tag = config.config_string(dat, [section, "tag"])
         return constellation.ImageReference(self.repo, name, tag)
+
+
+def config_value(data, path, data_type, is_optional, default=None):
```

Comment:
This method should live in `constellation` - PR for that here: https://github.com/reside-ic/constellation/pull/18